### ISSUE TITLE
Give all models id context

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#443]
Many models do not include a prefix to their id to give context. For example, a FixModel has the id of `BaseModel-19`. They should have names from which we can infer their class.

![image](https://cloud.githubusercontent.com/assets/5103735/25828049/2353664e-341c-11e7-8378-30d8e75ff796.png)

**Comment by n8rzz** [comment]
yep, the `BaseModel` part was just implemented.  so we just need to update the inheriting classes to pass a string in the `super()` call inside the constructor.
